content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -9,4 +9,339 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-This page is under construction.
+Ambient splits Istio’s functionality into two distinct layers, a secure overlay layer and a Layer 7 processing layer.
+The waypoint proxy is an optional component that is Envoy-based and handles L7 processing for different resources.
+What is unique about the waypoint proxy is that it runs outside of the application pod. A waypoint proxy can install,
+upgrade, and scale independently from the application, as well as reduce operational costs. When deploying a waypoint,
+you can configure the waypoint to process traffic for different resource types such as `service`, `workload`, or `all`.
+
+## Do you need a waypoint proxy?
+
+This layered approach of ambient allows users to adopt Istio in a more incremental fashion, smoothly transitioning from no mesh, to the secure overlay, to full L7 processing. If your applications require any of the following L7 mesh functions, you will need to use waypoint proxy for your applications:
+
+{{< image width="100%"
+link="L7-processing-layer.png"
+caption="L7 processing layer"
+>}}
+
+## Deploy a waypoint proxy
+
+Waypoint proxies can process traffic for `service`, `workload`, or `all`. You can also configure your waypoint proxy to
+process `none` of the traffic, which is primarily used for testing purpose as you incrementally add a waypoint proxy to
+your application.
+
+Before you deploy a waypoint proxy for a specific namespace, confirm the namespace is labeled with `istio.io/dataplane-mode: ambient`:
```

Comment:
I would highly recommend users to label ns ambient first. I think it is not required but it is where things most tested.

Reply:
right now the waypoint **must** be the the same namespace as the svc/wl it will serve. longer term we would like to be able to have reference grants allowing cross namespace waypoint use but that will not be in this release.